server/@types/models/Course.ts

```diff
@@ -4,6 +4,7 @@ import type { CoursePrerequisite } from './CoursePrerequisite'
 export type Course = {
   id: string
   name: string
+  alternateName: string
```

Comment:
Sorry, I should have pointed this out on the ticket, but this can be optional, as some courses don't have acronyms e.g. `iHorizon`.